File: from-2801-to-2900/2843-count-symmetric-integers/solution.go
func countSymmetricIntegers(low int, high int) int {
    result := 0
    for c := 11; c <= 99; c += 11 {
        if c >= low && c <= high {
            result += 1
        }
    }
    c := 1000
    for c <= high {
        if c >= low {
            if (c / 1000) + ((c /100) % 10) == c % 10 + (c % 100) / 10 {
                result += 1
            }
        }
        c += 1
    }
    return result
}

File: from-2801-to-2900/2843-count-symmetric-integers/solution.py
class Solution:
    def countSymmetricIntegers(self, low: int, high: int) -> int:
        result = 0
        for num in range(11, 100, 11):
            if low <= num <= high:
                result += 1

        for num in range(1000, high + 1):
            if num >= low:
                if (num // 1000) + ((num // 100) % 10) == num % 10 + (num % 100) // 10:
                    result += 1

        return result